docs/source/pre_application/cloud_node_management.rst

```diff
@@ -79,71 +65,47 @@ Now NuCypher Ops is installed we can create a droplet on Digital Ocean:
 
 .. code-block:: bash
 
-    nucypher-ops nodes create --network ibex --count 1 --cloudprovider digitalocean
+    nucypher-ops nodes create --network mainnet --count 1 --cloudprovider digitalocean
 
 At this point you should see the droplet in your Digital Ocean dashboard.
 Now we can deploy the PRE Node:
 
 .. code-block:: bash
 
-    nucypher-ops ursula deploy --eth-provider $INFURA_RINKEBY_URL --nucypher-image nucypher/nucypher:experimental --payment-provider $INFURA_MUMBAI_URL --network ibex
+    nucypher-ops ursula deploy --eth-provider $INFURA_MAINNET_URL --nucypher-image nucypher/nucypher:latest --payment-provider $INFURA_POLYGON_URL --network mainnet
```

Comment:
all of these args are optional.

`nucypher-ops ursula deploy` is all that is needed

Reply:
✔️